Source/Renderer/Context.js

```diff
@@ -195,7 +195,7 @@ define([
         webglOptions.alpha = defaultValue(webglOptions.alpha, false); // WebGL default is true
         webglOptions.stencil = defaultValue(webglOptions.stencil, true); // WebGL default is false
 
-        var defaultToWebgl2 = false;
+        var defaultToWebgl2 = defaultValue(options.defaultToWebgl2, false);
```

Comment:
I know this is internal for now, but this is going to percolate up to `Viewer` in time.  I think a better name would be `requestWebgl2` like the local below.